src/math/t/quaternion.lisp

```diff
@@ -0,0 +1,100 @@
+(define-test add-test
```

Comment:
Oh no.
We have a problem.
There already are tests with the same name, in `vector.lisp` and `matrix.lisp`.
And according to the job log, only `vector.lisp` test has actually run.

Please rename all quaternion tests to quaternion-<whatitdoes>-test, and the issue with clashing tests names I will address ASAP in the separate issue.